src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+# ---------------------------------------------------------
+# The Gradle build expands its project properties to
+# replace the substitution tokens.
+# ---------------------------------------------------------
+
+# shown on the actuator /info endpoint
+info:
+  app:
+    name: ${name}
+    description: ${description}
+    java:
+      source: ${sourceCompatibility}
+      target: ${targetCompatibility}
+
+# changes actuator endpoints - /manage/info
+management:
+  context-path: "/manage"
```

Comment:
This is the config file that is package inside the JAR.  The ${} are replaced by the build process using the Gradle project properties.